src/components/accessories/patientDataForm/PatientDataForm.tsx

```diff
@@ -98,6 +99,50 @@ const PatientDataForm: FunctionComponent<TProps> = ({
     }))
   );
 
+  const allAgeTypes = useSelector(
+    (state: IState) => state.ageTypes.getAllAgeTypes.data
+  );
+
+  const getBirthDateAndAge = (
```

Comment:
Sorry, I try to explain my self properly: can we move this method under the libraries folder? Like under formDataHandling that seems to me the right place to store this function.